app/models/project.rb

```diff
@@ -1,6 +1,7 @@
 class Project < ApplicationRecord
   has_many :votes
   has_many :users, through: :votes
+  belongs_to :user
```

Comment:
Not sure how I feel about having many `users` and belonging to a `user`. Should we change this line to:

```
belongs_to :owner , class_name: 'User', foreign_key: 'user_id'
```

If we do that, does it also make sense to change the users through the votes table to `:voters`?